04-shapes/Shapes.java

```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
+public class Shapes {
+  public static void main(String[] args) {
+    areaRectangle();
+    areaTriangle();
+    areaCircle();
+    areaTrapezoid();
+
+  }//end main method
+
+  public static void areaRectangle(){
+    Scanner input = new Scanner(System.in);
+    int length = 0;
+    int width = 0;
```

Comment:
if these values were passed into the method using parameters (`areaRectangle(int length, int width)`) i could see an advantage...you could reuse the methods later on when you determine volume.